src/flow.js

```diff
@@ -516,13 +516,9 @@ export default class flow {
     }
 
     if (!utils.directoryExists('./assets/')) {
-      utils.run('mkdir assets', false)
-        .then(() => {
-          processChanges()
-        });
-    } else {
-      processChanges()
+       utils.directoryCreate('assets');
```

Comment:
I could have dropped the `if` as well because `utils.directoryCreate` already does that too